common4j/src/main/com/microsoft/identity/common/java/providers/response.json

```diff
@@ -0,0 +1,11 @@
+  "refresh_token": "0.ASkAZ86Q6ltuFkKW35sM_4_IjsK4DUsmnxdEi94_DjZW-OApAMk.AgABAAAAAAD--DLA3VO7QrddgJg7WevrAgDs_wQA9P__R-sFbqD-hl8MyNcNtOd-qvH1GOXhKEyuF0tpY4-UCLi_l5ebSc7YGAs5GU_fxZPW7bySjFW8iMOrZzecMQRgeAZFIkHx8cx2OCYmIy8G1BV1mscBg8iLHzHNJx7ZsBDS90EKXtR6oFeFX6eaABUhc-DSr5w7U-zviUVhdfUb7zfmgQdjag2vdIU5mHLnADp21vF_Y7FfQFultlr1siJ0Jg7AXCkxnxhye7jnY9fu2mBDHsGWTiDmfk925CW2xPy15pxCkxYMq4OiJdiRtt7r6Jmp85RxeM1ZIQBMGwmvqBR10UL-18y2czsMY4xt6KUdB87Ah4CCUQ3e-MRcKtsP2Vjy1Mkw7H32ITrD3i6cr-o3DdL4W6DXnPDw-17k2nh5Bq0hx3ChlnuiBbizLrzkLFvQHNGUNo8TN8tsuM2rhZ9ONd2bnH7bESKOJCXawZObCx1sQ-9rhiA_5d8qBMh7z8UeoMZUhZNXYUhlvkQ2CZGWOkeAFWGcgMHhmQ822ebP_xhUjsKflRvjArV4ThF49Tz7AWnQ2zaod7p_hZ67bYzOioKiIwtR8rTvZnbM3iuFBRsRc35Dwhta565s8nVtUoiwOllyJTDqmG0-_ax600k5eiXqMfKU-w-UhmbWRPaTHmgzGfGnQO4KMgS-sK0oqkXjG3wlrPbsziL076Ns8aBzHXlILte-B7K3-QLHkRxwG8zR0BjIoptuLqp0P06k-DWSl5CZld0dan-7Us3F71nSS5FxsYVzHB-o2ceR1gImmXOjCwx8ISi0ztWwK5o_Ik_PpOwrgsb1Vr6erY14vSTDhBKB6WBW_TTOt1JGVl1UrRcAxaEiv07eS1I47alE_k_OvH6Npy_bT_9Pjz2dw0lSFXWKgLoW0DK5h-PK7LIlXK3DWQcFtkxWf101-M9sB5FQYBzY93qoORILDSh6FxkUF1WXub69fOShpL4n1Se9HoIBBtPg8aiAhShMi6HEp8gH2jxNq3o91WBsdRRR58vnpuHpKLhnx1Xc3zDyLQubHTDMumToaVdkP6Y5elSVuw",
```

Comment:
Looks like we have a token checked into this public repo here

Reply:
I'll remove them, thanks!